LICENSE

```diff
@@ -201,3 +201,10 @@
    limitations under the License.
 
 
+Copyright 2023 Metrological
```

Comment:
Thanks Chris:
Sorry, I didn't make this clear: Please use the generic form not specific year etc. so the same entry will satisfy all MIT needs. i.e.

```
MIT License
===========
Copyright <YEAR> <COPYRIGHT HOLDER>

Permission is...
```